SETUP.md

```diff
@@ -69,3 +70,13 @@ Ensure that you have docker and docker-compose installed on your machine, as wel
   - Create an account on snyk.io
   - Run 'snyk auth' in a terminal
   - Login when prompted (either in using the provided URL or in the opened browser)
+
+* Accessibility Check Tool (pa11y)
+
+  - Run 'run 'npm install -g pa11y' in a terminal
```

Comment:
Is there a way that just adds this to our package.json? Why does each person need to do an install?

Reply:
The way you are using it now doesn't require adding it to package.json so we can leave this as is for now.

Reply:
As discussed offline, I tried to installed it inside the front end container using _**yarn add pa11y**_, since _**npm install pa11y**_ gives me warnings and errors, but I wasn't able to use it, meaning that I wasn't even able to execute a _**pa11ly --version**_.

As agreed, we'll keep it locally for now.